cli_controller.py

```diff
@@ -0,0 +1,109 @@
+from command_node import CommandNode
+import json
+
+# The controller for the commandline interface
+class CLIController:
+    # Files given have to be JSON files.
+    def __init__(self, tmp_command_file_list=None):
```

Comment:
`tmp_command_file_list` -> `command_file_list`
you should drop the prefix

Reply:
Resolved in commit 
https://github.com/R2D2-2019/swarm_controller/pull/5/commits/ce29865273897da289128759a5e7650c1ea05dc1